src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"Black", "White", "Yellow", "Green", "Orange", "Blue"};
```

Comment:
what about creating an enum and using it it other classes? It will be better instead of the strings you write in both ColorSupplier and FigureSupplier classes

Reply:
Why then at the task said  that the method shoud return String: "For this purpose create two more classes:

ColorSupplier with public String getRandomColor() method - for generating random color,"